Feature/renovate UI variables

boddissattva opened the pull request.

iguissouma committed:
feature(ui): init an nx module

iguissouma committed:
feature(ui): configure /new/** ui

iguissouma committed:
feature(ui): add a link to try new ui

iguissouma committed:
feature(ui): add ui-material module

iguissouma committed:
feature(ui): add first ui draft

iguissouma committed:
feature(ui): update storybook for execution badge

iguissouma committed:
feature(ui): add execution badge status to scenarios list

iguissouma committed:
feature(ui): use mat table with flex box

iguissouma committed:
feature(ui): add covalent dependency

iguissouma committed:
feature(ui): add delete confirm dialog

iguissouma committed:
feature(ui): add delete action

iguissouma committed:
feature(ui): add delete action

iguissouma committed:
feature(ui): add some scenarios filter/sort logic

boddissattva committed:
fix(nx): Build ok

iguissouma committed:
feature(ui): refactor scenarios filter/sort logic into a component

iguissouma committed:
feature(ui): add login/logout logic

iguissouma committed:
feature(ui): update e2e test

iguissouma committed:
feature(ui): preserve scenarios filter state

iguissouma committed:
feature(ui): add auth cypress spec #wip

iguissouma committed:
feature(ui): add no data row when scenario result is empty

iguissouma committed:
feature(ui): add mat-toggle-visibility for password field

iguissouma committed:
feature(ui): add simple typed translation sample

iguissouma committed:
feature(ui): rework ui layout

iguissouma committed:
feature(ui): filter scenarios by text looking in all object properties

iguissouma committed:
feature(ui): add sort and pagination

iguissouma committed:
feature(ui): fix undefined this in filters.ts

smaouiom committed:
wip main layout design

iguissouma committed:
feature(ui): add scenario text view page

smaouiom committed:
wip main layout design: make assets and styles as libraries

smaouiom committed:
wip main layout design: add translation and account components

smaouiom committed:
wip main layout design: rework header design

smaouiom committed:
wip main layout design: change logo

iguissouma committed:
feature(ui): reformat code

iguissouma committed:
feature(ui): add scenario text run page #wip

iguissouma committed:
feature(ui): add scenario text run page #wip

iguissouma committed:
feature(ui): add scenario text run page #wip
rework observables

iguissouma committed:
feature(ui): add scenario text run page #wip
rework observables

iguissouma committed:
feature(ui): add scenario text run page #wip
rework with dotProp

iguissouma committed:
feature(ui): add scenario text run page #wip
format

iguissouma committed:
feature(ui): add simple translation switch

iguissouma committed:
feature(ui): show header icons sm screen

iguissouma committed:
feature(ui): add scenario text run page #wip
add execution icon info

iguissouma committed:
feature(ui): add scenario text edit page #wip

iguissouma committed:
feature(ui): rework theme

iguissouma committed:
feature(ui): add few animations

iguissouma committed:
feature(ui): toggle theme light/dark

iguissouma committed:
feature(ui): add page header component

iguissouma committed:
feature(ui): save scenario

iguissouma committed:
feature(ui): add more css helpers

iguissouma committed:
feature(ui): try page header

iguissouma committed:
feature(ui): reformat

iguissouma committed:
feature(ui): default options for snackbar

iguissouma committed:
feature(ui): remove appBanner style

iguissouma committed:
feature(ui): redesign page header based on theme

iguissouma committed:
feature(ui): add MatTooltip module

iguissouma committed:
feature(ui): rework page headers

iguissouma committed:
feature(run): add console to show report output #wip

iguissouma committed:
feature(ui): add run fab button

iguissouma committed:
feature(ui): refine run scenario

iguissouma committed:
feature(ui): refine edit scenario

iguissouma committed:
feature(ui): add edit link

iguissouma committed:
feature(ui): add first schema support #wip

iguissouma committed:
chore(ui): update to Nx11

iguissouma committed:
feature(ui): update schema

iguissouma committed:
feature(ui): update schema

iguissouma committed:
feature(ui): reformat

iguissouma committed:
feature(ui): extract sidemenu component

iguissouma committed:
feature(ui): add resume/pause button for scenario

iguissouma committed:
feature(ui): add targets mat menu #wip

iguissouma committed:
feature(ui): add targets mat menu #wip

iguissouma committed:
feature(ui): add campaigns library

iguissouma committed:
feature(ui): add campaigns page #wip

iguissouma committed:
feature(ui): add campaign view page #wip

iguissouma committed:
feature(ui): add campaign view page #wip

iguissouma committed:
feature(ui): add campaign view page #wip

iguissouma committed:
feature(ui): add campaign view page #wip

iguissouma committed:
feature(ui): add campaign run page #wip

iguissouma committed:
feature(ui): add campaign run page #wip

iguissouma committed:
feature(ui): add campaign run page #wip

iguissouma committed:
feature(ui): add campaign stop #wip

iguissouma committed:
feature(ui): rework execution badge

iguissouma committed:
feature(ui): add delete campaign
add edit page #wip

iguissouma committed:
feature(ui): add edit page #wip

iguissouma committed:
feature(ui): rework edit/save page #wip

iguissouma committed:
feature(ui): rework campaign edit/save page #wip

iguissouma committed:
feature(ui): rework campaign edit/save page #wip

iguissouma committed:
feature(ui): hack to fix auth first value emit is null #review

iguissouma committed:
feature(ui): change view icon

iguissouma committed:
feature(ui): change width id

iguissouma committed:
feature(ui): rework i18n #wip

iguissouma committed:
feature(ui): add logout #wip

iguissouma committed:
feature(ui): add backIcon to page-header-component #wip

iguissouma committed:
feature(ui): put fab-button on page header

iguissouma committed:
feature(ui): fix style

nbrouand committed:
Update release.yml

nbrouand committed:
Update release.yml

nbrouand committed:
Create test.yml

nbrouand committed:
Update test.yml

nbrouand committed:
Update test.yml

nbrouand committed:
Update test.yml

iguissouma committed:
chore(ui): fix linter errors #wip

iguissouma committed:
chore(ui): fix storybook styles

nbrouand committed:
Merge branch 'master' into master

boddissattva committed:
feat(ui): Add global variables library

boddissattva committed:
feat(ui): Add global variables to chutney menu

dependabot-preview[bot] committed:
chore(deps): Bump picocli from 4.2.0 to 4.6.1 (#319)

Bumps [picocli](https://github.com/remkop/picocli) from 4.2.0 to 4.6.1.
- [Release notes](https://github.com/remkop/picocli/releases)
- [Changelog](https://github.com/remkop/picocli/blob/master/RELEASE-NOTES.md)
- [Commits](https://github.com/remkop/picocli/compare/v4.2.0...v4.6.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

boddissattva committed:
feat(ui): Add global variables read

nbrouand committed:
Chore/refacto env (#325)

* chore(environment + glacio-adapter + server) Refacto environment

- Renaming
- Create configutation and spring configuration
- Remove optional
- Clean up code

* chore(environment + glacio-adapter + server): Remove some dependency in infra/api from enviornment domain

* chore(environment + server): Move TargetId to agent domain

boddissattva committed:
feat(ui): Add global variables delete

boddissattva committed:
feat(ui): Add global variables save

rbenyoussef committed:
feat(server): Improve dataset iterations (#336)

boddissattva committed:
feat(ui): Add global variables add

boddissattva committed:
feat(ui): Add global variables rename

boddissattva committed:
fix(ui): global variable last fixes

nbrouand committed:
refacto(glacio-adapter): Refacto to expose dto and not domain (#343)

nbrouand committed:
chore: Release 1.2.14

nbrouand committed:
chore: Prepare next development 1.2.15-SNAPSHOT

boddissattva committed:
feat(ui): global variables component/container distinction

boddissattva committed:
chore(ui): Actions into page header

boddissattva commented:
![Chutney - Mozilla Firefox 2021-02-17 11-15-09](https://user-images.githubusercontent.com/58807088/108191229-31dfcd00-7113-11eb-888e-148f4725482b.gif)

nbrouand commented:
When adding a new group, the json should be empty.

bessonm committed:
bugfix(ui): Missing hash in URLs on scenario exec page (#348)

iguissouma commented on server/src/main/java/com/chutneytesting/execution/domain/campaign/CampaignExecutionEngine.java:
```diff
@@ -61,8 +61,6 @@
     private final DataSetHistoryRepository dataSetHistoryRepository;
     private final JiraXrayPlugin jiraXrayPlugin;
     private final ChutneyMetrics metrics;
-    private ExecutorService asyncExecutor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
```
I added this to execute code asynchronously for new UI, was it removed?

iguissouma commented on nx/libs/ui-commons/src/lib/components/page-header/page-header.component.scss:
```diff
@@ -15,7 +15,7 @@
 
 chutney-page-header-action {
```
how it's shown for other pages?

iguissouma commented:
I'm not sure about multiple floating action buttons, I think its not compliant with material spec...
see the link:
https://material.io/components/buttons-floating-action-button#behavior

iguissouma commented:
Add popup to confirm delete

rbenyoussef committed:
feat(server/ui): Add jira link & filter on list (#349)

* feat(server+ui): add jira picto on campaign list & scenarios list
* feat(server+ui) : filter scenarios by jira test exec on campaign edition

bessonm committed:
doc: Update Readme with Kotlin DSL & Discussions (#353)

* doc: Update Readme with Kotlin DSL & Discussions

* typo

rbenyoussef committed:
bugfix(task-impl): infinite loop on Selenium ZK by (#351)

nbrouand committed:
chore: Release 1.2.15

nbrouand committed:
chore: Prepare next development 1.2.16-SNAPSHOT

iguissouma committed:
chore(ui): bump deps

iguissouma committed:
chore(ui): fix css z-index

96RadhikaJadhav committed:
chore: Add bug, feature and PR templates (#350)

Co-authored-by: bessonm <besson.mael@gmail.com>

rbenyoussef committed:
feat(ui): preview scenario error on campaign report (#356)

dependabot-preview[bot] committed:
chore(deps): Bump build-helper-maven-plugin from 3.0.0 to 3.2.0 (#357)

Bumps [build-helper-maven-plugin](https://github.com/mojohaus/build-helper-maven-plugin) from 3.0.0 to 3.2.0.
- [Release notes](https://github.com/mojohaus/build-helper-maven-plugin/releases)
- [Commits](https://github.com/mojohaus/build-helper-maven-plugin/compare/build-helper-maven-plugin-3.0.0...build-helper-maven-plugin-3.2.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

dependabot-preview[bot] committed:
chore(deps-dev): Bump activemq.version from 5.16.0 to 5.16.1 (#359)

Bumps `activemq.version` from 5.16.0 to 5.16.1.

Updates `activemq-broker` from 5.16.0 to 5.16.1
- [Release notes](https://github.com/apache/activemq/releases)
- [Commits](https://github.com/apache/activemq/compare/activemq-5.16.0...activemq-5.16.1)

Updates `activemq-junit` from 5.16.0 to 5.16.1

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

dependabot-preview[bot] committed:
chore(deps): Bump mongodb.version from 3.8.0 to 3.12.8 (#358)

Bumps `mongodb.version` from 3.8.0 to 3.12.8.

Updates `mongodb-driver` from 3.8.0 to 3.12.8
- [Release notes](https://github.com/mongodb/mongo-java-driver/releases)
- [Commits](https://github.com/mongodb/mongo-java-driver/compare/r3.8.0...r3.12.8)

Updates `mongodb-driver-core` from 3.8.0 to 3.12.8
- [Release notes](https://github.com/mongodb/mongo-java-driver/releases)
- [Commits](https://github.com/mongodb/mongo-java-driver/compare/r3.8.0...r3.12.8)

Updates `bson` from 3.8.0 to 3.12.8
- [Release notes](https://github.com/mongodb/mongo-java-driver/releases)
- [Commits](https://github.com/mongodb/mongo-java-driver/compare/r3.8.0...r3.12.8)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

dependabot-preview[bot] committed:
chore(deps): Bump maven-surefire-plugin from 3.0.0-M4 to 3.0.0-M5 (#360)

Bumps [maven-surefire-plugin](https://github.com/apache/maven-surefire) from 3.0.0-M4 to 3.0.0-M5.
- [Release notes](https://github.com/apache/maven-surefire/releases)
- [Commits](https://github.com/apache/maven-surefire/compare/surefire-3.0.0-M4...surefire-3.0.0-M5)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

rbenyoussef committed:
feat(task-impl): allow add headers to http soap task (#363)

nbrouand committed:
fix(ui): remove double dash when selecting an execution

fixes https://github.com/chutney-testing/chutney/pull/366

nbrouand committed:
chore: Release 1.2.16

nbrouand committed:
chore: Prepare next development 1.2.17-SNAPSHOT

nbrouand committed:
chore(): Upgrade to jdk11 (#316)

nbrouand committed:
chore: Release 1.2.17

nbrouand committed:
chore: Prepare next development 1.2.18-SNAPSHOT

nbrouand committed:
chore: Add clean to maven release to avoid conflicts

nbrouand committed:
chore: Release 1.2.18

nbrouand committed:
chore: Prepare next development 1.2.19-SNAPSHOT

bessonm committed:
chore: Setup jdk11 config with nix (#370)

nbrouand committed:
feat(server): Configuration for ldap with only TLS1.1 (#372)

nbrouand committed:
bugfix(task-impl): Redo commit f61bf42733271a67c17c266e00df77cce9be3207 accidently reverted (#374)

nbrouand committed:
bugfix(server): Add awaitibility for test and Fix Step unit Test (#375)

fixes https://github.com/chutney-testing/chutney/pull/375

nbrouand committed:
chore: Release 1.2.19

nbrouand committed:
chore: Prepare next development 1.2.20-SNAPSHOT

bessonm committed:
chore: Fix pkg src generation (#377)

nbrouand committed:
bugfix: forget to remove unused autowired field (#380)

nbrouand committed:
chore: Release 1.2.20

boddissattva committed:
Chore/fix test timeout (#378)

* chore(): Delete bintray processing

* chore(): Output generated sources of cxf-xjc-plugin to jaxb directory

* chore(): Cache local hosts name and ips

* chore(): Revert timeout explosion in tests

nbrouand committed:
chore(engine+server+task-impl): Use awaitility + tools class instead of thread sleep (#382)

nbrouand committed:
chore: Prepare next development 1.2.21-SNAPSHOT

iguissouma committed:
chore(ui): preserve tree state

iguissouma committed:
chore(ui): add version field to scenario

boddissattva committed:
chore(): build ok after rebase
